guide/content/deployment-steps.md

```diff
@@ -13,13 +13,15 @@ description: Deployment steps
 * **PrincipalArn**: `arn:aws:iam::123456789012:root` (Amazon Resource Name of the principal that can assume the role.
 * **pEnableCloudTrial**: `true` (Enables CloudTrail at the organization level. Default is false.)
```

Comment:
Variable was fixed to `pEnableCloudTrail` elsewhere but still `pEnableCloudTrial` here.

Reply:
fixed.